PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1627,6 +1691,22 @@ struct HfTrackIndexSkimCreator {
     }
   }
 
+  /// Method to perform ML selections for 2-prong candidates after the rectangular selections
+  /// \param featuresCand is the vector with the candidate features
+  /// \param outputScores is the vector with the output scores to be filled
+  /// \param isSelected ia s bitmap with selection outcome
+  void is2ProngSelectedMlForHfFilters(std::vector<float> featuresCand, std::vector<float>& outputScores, int& isSelected)
```

Comment:
`is...` indicates that the function returns `bool`.
```suggestion
  void applyMlSelectionForHfFilters2Prong(std::vector<float> featuresCand, std::vector<float>& outputScores, int& isSelected)
```

Reply:
Done in https://github.com/AliceO2Group/O2Physics/pull/5430/commits/87b013b121005169bdec56960b9f53ee73e735c4